Modules/CalcOffence-3_0.lua

```diff
@@ -388,6 +388,14 @@ function calcs.offence(env, actor, activeSkill)
 			breakdown.AreaOfEffectMod = breakdown.mod(skillCfg, "AreaOfEffect")
 		end
 	end
+
```

Comment:
I'd say remove these extra blank gaps between this if statement and the ones above and below it. They're unnecessary besides looking better when you're editing this file, and fewer lines in the file is better.

Reply:
I think it makes it easier to read, and 2 blank lines isn't really a big deal when the file is already 2000+ lines long. But if people feel strongly about it I'll remove not, it's not a big deal.